lib/setup.test.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+// configures a hapi server that can be used
+// for unit testing outside of docker
+
+const hapi = require('hapi');
+const ObjectID = require('mongodb').ObjectID;
+const mongo = require('mockgo');
+const get = require('../methods/get.js').method;
+const track = require('../methods/track.js').method;
+const extractInfo = require('../methods/extractInfo.js').method;
+const types = require('../methods/types.js').method;
+
+module.exports = (options, callback) => {
+  const server = new hapi.Server({});
```

Comment:
My thinking was to eliminate rapptor as a possible source for problems, so that this will unit test the actual code for the server routes (in isolation of any rapptor/docker issues).  I can change this to bootstrap the server with rapptor instead if you want.